### PR TITLE
Adding deprecation notice to repo

### PR DESCRIPTION
**Pull request by mattfarina**

### CODE REVIEW

**Review by scottrigby — APPROVED** [review]
👍

**Comment by scottrigby** [comment]
See #23944

**Comment by sfernandez** [comment]
i'm astonished with the little time of migration do you deliver to applications using this chart.
a 1 year time-to-migrate-or-die with a global pandemic in place it's not sufficient.

Thank you guys for the systems i have to quick fix now.

**Comment by scottrigby** [comment]
@sfernandez I hope since this comment you were able to resolve your systems. The charts repo maintainers and the entire Helm team have worked hard to ensure end users will have to make only minimal changes to keep their systems running:

- There was a 1 year timeline, corresponding to match the Helm 2 deprecation announcement and timeline. The initial notice was added to this repo's README to clarify this [in November 2019](https://github.com/helm/charts/pull/19138)
- There was a period of time where we were unsure where charts package history would live. As it turns out, we collaborated with GitHub who generously offered to [sponsor hosting these indefinitely](https://helm.sh/blog/helm-turns-five/)
- I'm sure there are things we could have done better. However, if you want to see the step-by-step action plan to ensure the best experience possible for end users, including how to ensure charts will continue to be available to running systems with only minimal changes (the repo URL), see https://github.com/helm/charts/issues/23850, https://github.com/helm/charts/issues/23944, and the linked issues/PRs
- For a summary of the ongoing initiative to relocate charts to new repos, see: https://github.com/helm/charts/issues/21103#issuecomment-732992002. Chart relocation is not complete because maintaining charts has always been a community effort. Deprecated charts that have not yet relocated can always be adopted and un-deprecated. If you're interested in helping that effort, that issue is the entry point for – and place to track – participation and status.
 
To summarize that last comment link, Helm community charts development has not ended, it is simply bigger than ever before and requires new, distributed hosting, discovery, and tooling.

For additional history and information, these blog posts are good places to start:
- https://helm.sh/blog/charts-repo-deprecation/
- https://helm.sh/blog/helm-2-becomes-unsupported/

**Comment by sfernandez** [comment]
Hi @scottrigby ,

1 year announcement it is still insufficient to migrate an infrastructural service, also not providing a fallback/or an upgrade manual in the same notice. This worsens because of the COVID situation, in this context you should had given more time to people.

I will still use Helm because it's an amazing tool. But things could be done better.

Thank you for your time answering and your invitation to participate, i will have a look.
